client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -373,30 +373,50 @@ export default React.createClass( {
 
 	unverifiedDomainsCanManage() {
 		const domains = this.getDomains().filter( domain => domain.isPendingIcannVerification && domain.currentUserCanManage );
+		const severity = domains.some( ( { registrationMoment } ) =>
+			registrationMoment && moment( registrationMoment )
+				.add( 2, 'days' )
+				.isAfter( moment() ) ) ? 'is-info' : 'is-error';
 
 		if ( domains.length === 0 ) {
 			return null;
 		}
 
 		if ( domains.length === 1 ) {
-			const domain = domains[ 0 ].name,
+			const domain = domains[ 0 ].name;
+			let fullMessage, compactMessage;
+			if ( severity === 'is-error' ) {
 				fullMessage = this.translate(
 					'Your domain {{strong}}%(domain)s{{/strong}} may be suspended because your email address is not verified.',
 					{
 						components: { strong: <strong /> },
 						args: { domain }
-					} ),
+					} );
+					compactMessage = this.translate(
+						'Issues with {{strong}}%(domain)s{{/strong}}.',
+						{
+							components: { strong: <strong /> },
+							args: { domain }
+						} );
+			} else if ( severity === 'is-info' ) {
```

Comment:
This could just be an `else` since theres only two possible values.
Using `isWithinTwoDays` would also simplify this